freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/flow/FreeIpaPostInstallService.java

```diff
@@ -80,11 +80,11 @@ public void postInstallFreeIpa(Long stackId) throws Exception {
 
     private void modifyAdminPasswordExpirationIfNeeded(FreeIpaClient client) throws FreeIpaClientException {
         Optional<User> user = client.userFind(freeIpaClientFactory.getAdminUser());
-        if (user.isPresent() && !FreeIpaClient.PASSWORD_EXPIRATION_DATETIME.equals(user.get().getKrbPasswordExpiration())) {
+        if (user.isPresent() && !FreeIpaClient.MAX_PASSWORD_EXPIRATION_DATETIME.equals(user.get().getKrbPasswordExpiration())) {
```

Comment:
So the admin password is our own freeipa admin user? and this is to ensure that the password never expires?

Reply:
yes. otherwise the FMS won't be able to sync users or set passwords after some time in the future.